test/unit/specs/addressbook.spec.js

```diff
@@ -12,8 +12,78 @@ describe("Addressbook", () => {
     Addressbook = require("src/main/addressbook.js")
 
     jest.mock("axios", () => ({
-      get: async () => {
-        return { data: { result: { peers: [] } } }
+      getNetInfo: async () => {
```

Comment:
imo a lot of clutter here, that is not interesting for the test. what do you think?